src/components/util/ajax.js

```diff
@@ -99,11 +167,11 @@ export function promise(url, method = 'GET', headers = {}, data = null, cache =
             resolve(response);
         };
         xhr.onerror = reject;
-        if (cache === false && url.indexOf('format=json') > -1){
+        if (cache === false){
```

Comment:
This probably worth removing and then maybe setting Cache headers if cache is explicitly true. Idk. To consider later. The current version of this doesn't work anyways (might get "/some/path/&ts=..." (not "?"). Is unrelated to this branch so will save for later.